### PR TITLE
Use NOTION_RELEASE environment value if defined as version and sh compat

### PR DESCRIPTION
**Pull request by knixeur**
~Use technique suggested in https://stackoverflow.com/q/7292584
to check git can be executed~

Edit: I've updated the PR, we talked in IRC and an environment value is a better fit

### CODE REVIEW

**Comment by knixeur** [comment]
You're right, I'm swallowing the version it should go to the else condition too.

**Comment by raboof** [comment]
The 'else' expects you to be in a directory that follows the github tag zip download naming conventions - if there is a .git directory present then it's unlikely the directory name follows that convention...

File: version.sh [hunk 1]
```diff
@@ -20,10 +20,15 @@
 
 set -e
 
+if [ ! -z "$NOTION_RELEASE" ]; then
+    echo "$NOTION_RELEASE"
+    exit
```
Comment (raboof) [inline version.sh]: Can we use `elif` instead?

File: version.sh [hunk 2]
```diff
@@ -20,10 +20,15 @@
...
+fi
+
 if [ -e ".git" ]; then
   # Git:
   echo -n `git describe`
-  if [[ -n $(git status -s) ]]; then
+  if [ ! -z "$(echo -n `git status -s`)" ]; then
```
Comment (raboof) [inline version.sh]: Nice :+1:

File: version.sh (hunk 1 above)
Comment (knixeur) [reply version.sh]: Sure, done :)

**Pull request merged**